jenkins/run-server-func-tests

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+export EXTRA_PODMAN_SWITCHES=${EXTRA_PODMAN_SWITCHES:-"--pull=always -e=COV_REPORT_XML=yes"}
+export IMAGE_KIND=${IMAGE_KIND:-"fedora"}
+export IMAGE_REPO=${IMAGE_REPO:-"quay.io/pbench"}
+export IMAGE_ROLE=${IMAGE_ROLE:-"ci"}
+
+export PB_CONTAINER_REG=${PB_CONTAINER_REG:-"images.paas.redhat.com"}
+export PB_REPO_NAME=${PB_REPO_NAME:-"pbench"}
+
+export PB_ELASTICSEARCH_IMAGE_NAME=${PB_ELASTICSEARCH_IMAGE_NAME:-"elasticsearch"}
+export PB_ELASTICSEARCH_IMAGE_TAG=${PB_ELASTICSEARCH_IMAGE_TAG:-"7.13.3"}
+export PB_POSTGRESQL_IMAGE_NAME=${PB_POSTGRESQL_IMAGE_NAME:-"postgresql-13"}
+export PB_POSTGRESQL_IMAGE_TAG=${PB_POSTGRESQL_IMAGE_TAG:-"latest"}
+export PB_SERVER_IMAGE_NAME=${PB_SERVER_IMAGE_NAME:-"pbench-server-ci"}
+export PB_SERVER_IMAGE_TAG=${PB_SERVER_IMAGE_TAG:-"main"}
+export PB_POD_NAME=${PB_POD_NAME:-"pbench-server_${PB_SERVER_IMAGE_TAG}"}
+
+export PB_COMMON_IMAGE_PULL_POLICY=${PB_COMMON_IMAGE_PULL_POLICY:-"Always"}
+export PB_ELASTICSEARCH_IMAGE_PULL_POLICY="${PB_ELASTICSEARCH_IMAGE_PULL_POLICY:-${PB_COMMON_IMAGE_PULL_POLICY}}"
+export PB_POSTGRESQL_IMAGE_PULL_POLICY="${PB_POSTGRESQL_IMAGE_PULL_POLICY:-${PB_COMMON_IMAGE_PULL_POLICY}}"
+export PB_SERVER_IMAGE_PULL_POLICY="${PB_SERVER_IMAGE_PULL_POLICY:-${PB_COMMON_IMAGE_PULL_POLICY}}"
+
+export PB_DASHBOARD_DIR="${PB_DASHBOARD_DIR:-${PWD}/dashboard/build/}"
+
+function cleanup {
+    podman pod rm --force --ignore ${1} || true
+    podman container prune -f || true
+    podman container cleanup --all --rm || true
+    podman image prune -f
```

Comment:
Should line 30 have the `|| true` appendage as well?  (Now that we're no longer in the Groovy stage, it probably doesn't matter, but, since we really don't care what the exit status is, we might as well suppress the error...and it looks funny without it after the preceding three lines....)

Reply:
See PR #3112.